src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -166,14 +226,48 @@ public static int getChannelsNumberFromLibMediaInfo(String mediaInfoValue) {
 		}
 	}
 
+	public static int getBitsperSampleFromLibMediaInfo(String mediaInfoValue) {
+		if (isEmpty(mediaInfoValue)) {
+			LOGGER.warn("Empty value passed in. Returning default number 16.");
+			return 16;
+		}
+
+		// examples of libmediainfo  (mediainfo --Full --Language=raw file):
+		// Bit depth : 16
+		// Bit depth : 24
+		// Bit depth : / 24 / 24
+
+		int result = -1;
+		Matcher intMatcher = intPattern.matcher(mediaInfoValue);
+		while (intMatcher.find()) {
```

Comment:
@Sami32 I'm not sure if you're aware of this, but this will set the result to the *last* number in the string.